src/Sets/VPolygon.jl

```diff
@@ -301,57 +301,79 @@ const binary_or_brute_force = 10
 
 # return the index of a support vector of P along d from the list of vertices of P
 function _σ_helper(d::AbstractVector, P::VPolygon)
-    if length(P.vertices) > binary_or_brute_force
-        return _binary_support_vector(d, P)
+    vlist = P.vertices
+    return _σ_helper(d, vlist)
+end
+
+function _σ_helper(d::AbstractVector, vlist::Vector{VN}) where {N, VN<:AbstractVector{N}}
```

Comment:
i had to add this function so that it is cleaner to reuse `_σ_helper` in [this line](https://github.com/JuliaReach/LazySets.jl/pull/2488/files#diff-10c1b027a265435b398c6151f8e72799995bcff09e9a72d9d818b12683df9e98R335).